content/posts/02_staff_expectations/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "What is expected from staff engineers"
+date: 2022-12-18
+tags: [
+    "career", "growth"
+]
+---
+
+It's relatively easy to explain expectations for junior, mid, and even senior engineers. But it starts to get much harder at staff+ 
+positions. Career progression broadens the scope of work andresponsibilities. The environment and problems get more ambiguous. It's not easy
+to define a clear and measurable set of expectations, because staff roles also differ between companies and
+their actual needs. They can also vary between different [staff engineer archetypes](https://staffeng.com/guides/staff-archetypes).
+
+In the article, we will focus mostly on the **Tech Lead** archetype, which is the most popular in the industry. We will present some 
+expectations for staff engineers based on our experience. The list is not complete and there are many very important skills and 
+traits a staff engineer should have. We wanted to describe those that are fundamental regardless of the specific situation.
+
+# So what are those expectations?
+
+## Get things done
+![](./done.jpg)
+
+This might as well be applied to the earlier positions. But for staff engineers, it gets even more important. Staff engineers have to get 
```

Comment:
avoid regular gradation with `more`, `the most`, almost always there exists a dedicated phrase for what you're looking for: https://www.thesaurus.com/browse/more%20important

Reply:
I'm not sure those synonyms look better. Why would you avoid regular graduation? I see it might be nice to use some phrase in many cases, to show a refined, sophistacated language, but sometimes simplicity is better.